modules/local/generate_config.nf

```diff
@@ -0,0 +1,29 @@
+process GENERATE_CONFIG {
+    tag "$meta.id"
+    label 'process_single'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the GENERATE_CONFIG module. Please use docker or singularity containers."
+    }
+    container 'genomehubs/blobtoolkit:4.0.7'
+
+    input:
+    val meta
+
+    output:
+    tuple val(meta), path("${meta.id}/*.yaml") , emit: yaml
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    blobtoolkit-pipeline generate-config ${prefix}
```

Comment:
Is this a python script? Can it be used directly from `bin` rather than being accessed via the `blobtoolkit-pipeline` container? If we use the script directly, we can make the module `conda` compatible as well as make it light weight.

Not a priority change but something to think about.